src/pages/default/credentials/AddCredentialDetail.tsx

```diff
@@ -0,0 +1,128 @@
+import { Button } from "@/components/Button";
+import { BadgeNames, ItemAvatar } from "@/components/Item";
+import { ColorTheme } from "@/theme/colors";
+import { FlexLayouts, Layouts } from "@/theme/layouts";
+import { Typography } from "@/theme/typography";
+import React from "react";
+import { SafeAreaView, View, Text, StyleSheet } from "react-native";
+import Octicons from "@expo/vector-icons/Octicons";
+import { mockCredentials } from "@/services/mocks";
+
+const AddCredentialDetailScreen = ({ navigation, route }) => {
+  const credential = route.params.credential;
+  const navigateToAddCredentialOptions = () => {
+    navigation.navigate("AddCredentialOptions", { credential });
+  };
+  const navigateToAddCredentialDetail = (secondaryCredential) => {
+    navigation.push("AddCredentialDetail", { credential: secondaryCredential });
+  };
+
+  //TODO: Replace mockCredentials with real profile credentials
+
+  return (
+    <SafeAreaView>
+      <View style={Layouts.container}>
+        <View style={[Layouts.row, FlexLayouts.containerHorizontalCenter]}>
```

Comment:
Similar to my other comment(s) we should just use flexbox directly then rely on arbitrary composition of ad hoc objects